internal/pkg/service/buffer/worker/service/file_close.go

```diff
@@ -52,7 +52,9 @@ func (s *Service) closeFiles(slicesWatcher *activeSlicesWatcher, d dependencies)
 			return context.WithTimeout(context.Background(), 2*time.Minute)
 		},
 		TaskFactory: func(event etcdop.WatchEventT[model.File]) task.Fn {
-			return func(ctx context.Context, logger log.Logger) task.Result {
+			return func(ctx context.Context, logger log.Logger) (result task.Result) {
+				defer checkAndWrapUserError(&result.Error)
```

Comment:
Nerobil som to nejak globalne, kedze aj ked je to teraz vsade rovnake, casom sa moze zoznam user error pre jednotlive tasky lisit.